web/src/index.css

```
@@ -38,33 +38,33 @@
   --color-sidebar: oklch(0.18 0.02 55);
-  --color-sidebar-foreground: oklch(0.80 0.02 75);
+  --color-sidebar-foreground: oklch(0.8 0.02 75);
   --color-sidebar-border: oklch(0.26 0.02 55);
   --color-sidebar-active: oklch(0.75 0.16 65);
 }
 
 @layer base {
   body {
     font-family:
-      "Inter",
+      'Inter',
```

Stylelint (17.13.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)